### ISSUE TITLE
Bake postgres GPG key into images?

### ISSUE THREAD

**Issue by alxndrsn** [getodk/central#352]
When building `service.dockerfile`, the following message appears on stderr:
```
Warning: apt-key output should not be parsed (stdout is not a terminal)
```
It looks like it's triggered by:
https://github.com/getodk/central/blob/2eea364384e3f978a04c89ed2d8b262e2d0bdba0/service.dockerfile#L5-L8
The man page for `apt-key` reads:
```
add filename
    Add a new key to the list of trusted keys. The key is read from the 
    filename given with the parameter filename or if the filename is - from
    standard input.

    It is critical that keys added manually via apt-key are verified to
    belong to the owner of the repositories they claim to be for otherwise
    the apt-secure(8) infrastructure is completely undermined.

    Note: Instead of using this command a keyring should be placed directly
    in the /etc/apt/trusted.gpg.d/ directory with a descriptive name and 
    either "gpg" or "asc" as file extension.
```
Perhaps the `apt-key add` call can be replaced with:

1. copying https://www.postgresql.org/media/keys/ACCC4CF8.asc into e.g. `files/service/postgres-apt-key-ACCC4CF8.asc`
2. `COPY files/service/postgres-apt-key-ACCC4CF8.asc /etc/apt/trusted.gpg.d/`

**Comment by alxndrsn** [comment]
The postgres wiki page for apt (https://wiki.postgresql.org/wiki/Apt#Quickstart) recommends:

```
curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | gpg --dearmor | sudo tee /etc/apt/trusted.gpg.d/apt.postgresql.org.gpg >/dev/null
```
we may not required `sudo`, in which case:
```
curl https://www.postgresql.org/media/keys/ACCC4CF8.asc | gpg --dearmor > /etc/apt/trusted.gpg.d/apt.postgresql.org.gpg
```

**Comment by matthew-white** [comment]
Closing because I think this is resolved by #353. (Please reopen if there's still work to do!)
